package.json

```diff
@@ -332,4 +332,4 @@
 		"chokidar": "^3.5.1",
 		"eslint-nibble": "^6.1.0"
 	}
-}
\ No newline at end of file
+}
```

Comment:
nit: same here as the `client/lib/plans/features-list.js` file.

Reply:
@worldomonation  - I tried checking out the latest from package.json from trunk to overwrite my changes but it looks like when I did a "git pull origin trunk" these changes got into my fork. I am not able to  revert these file back to the state they were when i forked the branch